fix(ddx): content review and update (#2718)
scaleway/docs-content: dedibox-console/classic-hosting/how-to/configure-php.mdx

4. Click the **Edit** action.
    <Lightbox src="scaleway_website_php.webp" />
5. Select the PHP version of your choice.
6. Click **Submit** to update your PHP version.

## How to configure PHP 

<Message type="important">
  Configuring PHP is possible for all our offers. However, ensure you are running at least PHP version 5.3 and above.
</Message>

1. Upload a `.user.ini`file to the folder corresponding to your subdomain (folder `/www` for example).
    <Message type="note">
      The majority of options are configurable, except for those with an impact on the resources of the server.
    </Message>
2. You can modify all values with `PHP_INI_ALL` changeable option you can find by clicking this [link](https://www.php.net/manual/en/ini.list.php).
    <Message type="important">
      - chown: modifications of the owner of a file are prohibited. Files uploaded using PHP or FTP already belong to your user.
      - system/exec/popen: the execution of binaries or CGI files is not possible for security reasons.
      - ASP is not supported. The execution of CGI scripts (cgi-bin) and other programs is disabled.
    </Message>

> ## How to solve a 500 Error in the htaccess file

Reviewer: ```suggestion
## How to solve a HTTP Error 500 in the htaccess file
```
not sure, wdyt?

Author: ```suggestion
## How to solve an HTTP error 500 in the htaccess file
```

Author: I've added a second suggestion, just to fix a/an HTTP

You can get this error because your `.htaccess` file has errors, such as:

- Presence of unauthorized Directive,
- Presence of a syntax error,
- `.htaccess` File transferred in binary, instead of a transfer as text,
- Lack of a final blank line.

1. Rename the `.htaccess` file to `htaccessX.txt`to solve these errors.
2. Create an empty `.htaccess` file.
3. Add the lines one by one until you find the lines that caused the error 500.

> ## How to solve an Error in a PHP Script

Reviewer: ```suggestion
## How to solve an error in a PHP Script
```
